migration-scripts/tag_externalips_from_k8s.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: $0 <cluster-id>"
+  exit 1
```

Comment:
The rest of the file is indented with 4 spaces, and the other file is indented with 2. 

Could you run `shfmt -i 4 -w ./migration-scripts/*` (from [brew](https://formulae.brew.sh/formula/shfmt) or [here](https://github.com/mvdan/sh?tab=readme-ov-file#shfmt)) to normalize on one ?